### ISSUE TITLE
Training finished but my stylized images came out black 

### ISSUE THREAD

**Issue by noufali** [CompVis/adaptive-style-transfer#12]
Thank you for providing this! I was able to apply your pre-trained models on different images successfully on my Ubuntu 16.04 machine using CUDA 9.1 and CuDNN 7. It looked really beautiful. However, I ran into an issue when I trained the model on my own dataset of styles (10 images). It took around a week to finish and when I finally applied my trained model, the images came out black. There was no style at all. I have a few thoughts on what went wrong possibly but I would really appreciate if you can help point me in the right direction. 

This is what what I got in my terminal when the training finished (I stitched the image together so a bit of the middle part is missing): 

![error](https://user-images.githubusercontent.com/22034616/47858676-fc3c3c00-ddc2-11e8-8a69-c83e16653bc2.jpg)

At the top, it says "Your CPU supports instructions that this Tensorflow binary was not compiled to use" and at the bottom is says "Load Failed". My Tensorflow version is 1.7. Could this be the issue? I'm not quite sure what "Load failed" means or what it is talking about. I just wanna point out the issue now before I start training for another week. 

This is the stylized folder that came out all black: 

<img width="525" alt="screen shot 2018-11-01 at 10 52 28 am" src="https://user-images.githubusercontent.com/22034616/47859265-52f64580-ddc4-11e8-94e9-6b6cbc68c6d4.png">

**Comment by asanakoy** [comment]
You should have checked intermediate results during training, to debug it on earlier stages and not wait for one week. We have trained on crops 768x768, so you may have to tune learning rate and batch size, for example. You can also try doing one step of encoder-decoder and one step of discriminator instead of our adaptive scheme.

**Comment by asanakoy** [comment]
BTW, what style dataset did you use? Images have to be related in style.

**Comment by noufali** [comment]
@asanakoy Yup I know! I created my own style of graphic images. All containing similar colours/themes/shapes but different compositions. 

Thanks for the fast response! I will look into what you mentioned.
